src/backend/distributed/shared_library_init.c

```diff
@@ -2296,3 +2305,23 @@ IsSuperuser(char *roleName)
 
 	return isSuperuser;
 }
+
+
+static void
+CitusObjectAccessHook(ObjectAccessType access, Oid classId, Oid objectId, int subId,
```

Comment:
we generally add function comments to all but the most trivial functions

since this is a very non-trivial function, it would be good to add one